### ISSUE TITLE
Possible to get liquid page values in template

### ISSUE THREAD

**Issue by xubeisi** [inukshuk/jekyll-scholar#326]
hi, @inukshuk It's a awesome plugin you guys developed!  

Recently, I modified the template a little so they could automatically underline the defined authors. 

Such as
![image](https://user-images.githubusercontent.com/1790841/112034965-14849100-8b0d-11eb-8113-4a6bbea5c0b5.png)

[code example here site.scholar.first_names_team](https://github.com/stjudecab/stjudecab.github.io/blob/26bd30282e2b30750caa8c58524eb71c3e9dbe83/_layouts/bib.html#L20)

my goal is each page could use the same bib template but highlight different authors. 

However, it doesn't seems possible to pass page values to in template such as [_layouts/bib.html](https://github.com/stjudecab/stjudecab.github.io/blob/master/_layouts/bib.html). I tried read the code, seems only "entry" stored data. 

thus wondering for a query like 
```
{% bibliography -f {{ bibfile }} --template bib -q @*[year={{y}}]* %}
```
is it possible to pass some parameters like
```
{% bibliography -f {{ bibfile }} --template bib -q @*[year={{y}}]* --values "first_authors_to_highlight:Author1,Author2" %}
```

then in template.html, {{ site.scholar.first_authors_to_highlight }} would be "Author1,Author2"

Thank you!

**Comment by inukshuk** [comment]
Good question!

There's currently no tag parameter we specifically pass-through. However, you mentioned you would like set these parameters per page, so I was curious if the page's front-matter config is accessible by the templates. It currently isn't, but it would be easy to expose it (see commit above). Would that work for you?

**Comment by xubeisi** [comment]
> Good question!
> 
> There's currently no tag parameter we specifically pass-through. However, you mentioned you would like set these parameters per page, so I was curious if the page's front-matter config is accessible by the templates. It currently isn't, but it would be easy to expose it (see commit above). Would that work for you?

hi, Sylvester, that's great! I thought that's not possible thus I was seeking for solutions by pass variables. What you proposed were much better if it could be done. Thank you for considering that!!

**Comment by inukshuk** [comment]
The test passes fine, so this should work OK. I'll make a point release for you to try out.

**Comment by inukshuk** [comment]
Landed in 6.8.1

**Comment by xubeisi** [comment]
@inukshuk  Awesome, it works! thank you very much! I really appreciated your fast implementation!
